Documentation/admin/persistent-volumes.md

```diff
@@ -0,0 +1,123 @@
+ name` defined in the `StorageClass` it references.
+
+When Persistent Volumes are dynamically provisioned, the `StorageClass` fields `provisioner`, `parameters`, or `reclaimPolicy` may be used.
+
+Specify a default `StorageClass` for PVCs that don’t request a specific class.
+
+`StorageClass` objects cannot be updated once they are created.
+
+For more information, see [StorageClasses][storage-classes] in the Kubernetes documentation.
+
+### Defining Storage Classes for AWS
+
+Use kubectl to create an AWS StorageClass:
+
+```
+kubectl create -f
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: aws-ebs-sc
+provisioner: kubernetes.io/aws-ebs
+```
+
+### Defining Storage Classes for Azure
+
+Tectonic supports both Azure Disks and Azure Files.
+* Use Azure Files for dev and debugging tools that need access from many VMs.
+* Use Azure Disks for data accessed only from within the VM to which it is attached.
+
+For more information on Azure storage class types, see [When to use Azure Blobs, Azure Files, or Azure Disks][azure-blobs] in the Microsoft Azure documentation.
+
+Use kubectl to create an Azure Disk `StorageClass` of `kind: managed`:
+
+```
+kubectl create -f
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+   name: azure-disk-managed
+provisioner: kubernetes.io/azure-disk
+parameters:
+   storageaccounttype: Standard_LRS
+   kind: managed
+```
+
+## Defining Persistent Volumes
+
+Persistent Volumes may be defined statically for use across the cluster. Static Persistent Volume definitions must specify a `StorageClass`, which must exist before the Persistent Volume is defined.
+
+To define Persistent Volumes, go to *Administration > Persistent Volume*, and click *Create*. Enter the definition, and click *Create*.
+
+The following example creates a Persistent Volume which references an existing `StorageClass`, using `storageClassName: slow`.
+
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: EBSpv
+spec:
+  accessModes:
+  - ReadWriteOnce
+  awsElasticBlockStore:
+    fsType: ext4
+    volumeID: aws://ca-central-1b/vol-0c9
+  capacity:
+    storage: 8Gi
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: aws-ebs-sc
+```
+
+### Azure Persistent Volumes
+
+Tectonic uses Managed Disks for the VMs. As a result, Persistent Volume Claims (PVCs) using an Azure Disk storage class that has `kind: shared` (the default) will fail to mount to the container.
+
+To define PVCs on Azure, first create `StorageClasses` of `kind: managed`. Then, use those classes when creating dynamic or static Persistent Volumes.
+
+## Defining Persistent Volume Claims
+
+Persistent Volumes may also be defined dynamically, by creating a Persistent Volume Claim which references an existing `storageClassName`.
```

Comment:
To my understanding, a PVC uses some PV (by storage class name) as backing storage. PVCs are not a way to dynamically define PVs.

Reply:
to my understanding, PVCs are the way to dynamically define PVs.  https://kubernetes.io/docs/concepts/storage/persistent-volumes/#dynamic

Reply:
Ok. Good distinction. 

I think the phrasing should be more like "A PV may be dynamically created by the cluster for a PVC whose request does not match any existing statically-defined PV name. The cluster can create this dynamic PV when the PVC's request is for an existing `StorageClassName`.

Because a PVC may also just match a statically-defined PV name directly, and I'm not sure that it is clear these are two alternatives.

I think the document could use an intro paragraph that more general explains the relationship between PVC and the backing PV.